cmd/miniooni/main.go

```diff
@@ -166,26 +173,17 @@ func main() {
 		Level:   log.InfoLevel,
 		Handler: &logHandler{Writer: os.Stderr},
 	}
+	log.Log = logger // from now on logs may be redirected
```

Comment:
There is no log redirection on files anymore, so the command can go.

Reply:
`log.Log` redirect to `logHandler` with `InfoLevel`

Reply:
I don't understand what you mean here. Usually with "redirect" we mean that they are sent to a file, here we're not sending the logs to a file. They're still sent to stderr as with the default logger.

Reply:
We used multi to send logs to file.

Reply:
right, but that's not the case anymore, so the comment is outdated, no?